test/e2e/framework/framework.go

```diff
@@ -57,7 +58,8 @@ type Framework struct {
 	Clientset_1_2      *release_1_2.Clientset
 	Clientset_1_3      *release_1_3.Clientset
 
-	FederationClientset *federation_internalclientset.Clientset
+	FederationClientset     *federation_internalclientset.Clientset
```

Comment:
Will be good to add a TODO or a NOTE here to remove unversioned FederationClientset, so that anyone writing new code knows that they should use the versioned client set.